src/DynamoFactory.ts

```diff
@@ -1,7 +1,7 @@
-import { DynamoDB, DynamoDBClientConfig } from '@aws-sdk/client-dynamodb';
```

Comment:
I think you need to kill `aws-sdk/client-dynamodb` from devDependencies and peerDependencies as well

Reply:
corrected and updated the version like i was supposed to